turbo.json

```diff
@@ -1,13 +1,19 @@
 {
   "$schema": "https://turbo.build/schema.json",
   "tasks": {
+    "ready": {
+      "dependsOn": ["^ready"],
+      "outputs": ["dist/**", "build/**"]
+    },
     "dev": {
-      "dependsOn": ["^build"],
+      "dependsOn": ["^build", "ready"],
       "outputs": ["dist/**", "build/**"],
+      "env": ["__FIREFOX__"],
```

Comment:
Where is __DEV__ env, you're completely remove it, why?

Reply:
`cross-env __DEV__=true` is already exist at dev command in each packages.